src/action.ts

```diff
@@ -4,7 +4,7 @@ export interface RawAction {
     (... args: any[]): Promise<any> | void;
 }
 
-export default function action(actionType: string) {
+export default function action(actionType: string, options?: any) {
```

Comment:
I'm thinking the options should be of type { [key: string]: any }.

The reason being, there might be various middlewares that want options passed into them.  If you just pass (say) a string that middlewareA knows how to handle, and then later you want to consume middlewareB which requires an array of numbers, you'd have no way to support both.  By passing in a dictionary, each middleware can look for it's own option(s) based on it's own well-known key.